src/ViewHelpers/Format/SplitViewHelper.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file belongs to the package "TYPO3 Fluid".
+ * See LICENSE.txt that was shipped with this package.
+ */
+
+namespace TYPO3Fluid\Fluid\ViewHelpers\Format;
+
+use TYPO3Fluid\Fluid\Core\Rendering\RenderingContextInterface;
+use TYPO3Fluid\Fluid\Core\ViewHelper\AbstractViewHelper;
+use TYPO3Fluid\Fluid\Core\ViewHelper\Traits\CompileWithRenderStatic;
+
+final class SplitViewHelper extends AbstractViewHelper
+{
+    use CompileWithRenderStatic;
+
+    /**
+     * @var bool
+     */
+    protected $escapeChildren = false;
+
+    public function initializeArguments(): void
+    {
+        $this->registerArgument('delimiter', 'mixed', 'Delimiter string to explode with', true);
+        $this->registerArgument('value', 'string', 'The string to explode');
+        $this->registerArgument('limit', 'int', 'If limit is positive, a maximum of $limit items will be returned. If limit is negative, all items except for the last $limit items will be returned. 0 will be treated as 1.', false, PHP_INT_MAX);
+        $this->registerArgument('trim', 'bool', 'Trim items in array', false, false);
+        $this->registerArgument('removeEmpty', 'bool', 'Remove empty items from array. Items containing only whitespace are considered empty as well.', false, false);
```

Comment:
No need to overcomplicate stuff, there is no need for a splitter to remove empty stuff.

Reply:
Same here; if we would not have this parameter, integrators need to have one more f:if structure plus indentation to solve that. And I believe removing empty values could be a common use for this explosion.